src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private final RegistrationService registrationService = new RegistrationServiceImpl();
+
+    @Test
+    void register_nullUser_notOk() {
+        User nullUser = new User(null, null, null);
```

Comment:
The `User` class constructor should not allow for all `null` parameters. This test case suggests that it's possible to create a User with all `null` values, which should be prevented.